mongo/session.go

```diff
@@ -193,9 +194,12 @@ func (s *sessionImpl) WithTransaction(ctx context.Context, fn func(sessCtx Sessi
 			default:
 			}
 
-			if cerr, ok := err.(CommandError); ok {
-				if cerr.HasErrorLabel(driver.TransientTransactionError) {
-					continue
+			currErr := err
+			for ; currErr != nil; currErr = unwrap(currErr) {
```

Comment:
Is this the same as `errors.As(...)`, just for < Go 1.13?

Reply:
Yup! This is the workaround we add for the error helpers. It's basically what errors.As does